core/src/main/kotlin/br/com/concrete/yosef/api/component/ElementGroupComponent.kt

```diff
@@ -26,6 +28,7 @@ class ElementGroupComponent : Component {
 
     private val components: Map<String, DynamicPropertyCommand> = mapOf(
         ORIENTATION to OrientationCommand(),
+        GRAVITY to GravityCommand(),
```

Comment:
I understand why this is here, but I don't think this property is generic enough to be applied in this context. Unfortunately we assume the parent view group is a LinearLayout for now :/

Reply:
I agree, but since the library don't have any ways to deal with gravity i think it's a first step, my initial idea was to make a generic gravity property that would be apply to all components, but i couldn't do it :disappointed: also, I've tested it on a LinearLayout and it simple ignore the property